src/toolchain-conventions.adoc

```diff
@@ -203,6 +203,65 @@ RISC-V psABI has a contain sections to describe the code model:
 - https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#large-code-model[Large code model]: -mcmodel=large
 - https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#medium-position-independent-code-model[Medium position independent code model]:  -fpic, -fPIC, -fpie or -fPIE.
 
+== Specifying tuning processor and tuning features with -mtune
+
+Compilers accept the `-mtune` flag to specify the tuning processor model and
+features. They allow users to compile with the assumption of certain performance
+characteristics. Tuning processors and tuning features should not affect any of
+the _architectural_ properties or behaviors.
```

Comment:
I feel we might want to be more explicit here that tuning features must not change what cores the code is compatible with. I tried a few sentences but they all ended up sounding very legal-jargon-y, which I wanted to avoid (and more legal-jargon-y than your sentence here)